src/main/java/frc/robot/subsystems/BuddyBalanceSubsystem.java

```diff
@@ -4,107 +4,187 @@
+    rightMotor2 = MotorController.constructMotor(MotorConfig.BuddyBalanceRight2);
+    leftMotor1 = MotorController.constructMotor(MotorConfig.BuddyBalanceLeft1);
+    leftMotor2 = MotorController.constructMotor(MotorConfig.BuddyBalanceLeft2);
+    deployServo1 = new Servo(deployServo1ID);
+    deployServo2 = new Servo(deployServo2ID);
+    encoder = AbsoluteEncoder.constructREVEncoder(EncoderConfig.BuddyBalance);
+    unifiedPIDController = new PIDController(kDefaultMotorP, kDefaultMotorI, kDefaultMotorD);
+
+    tunerNumP = new TunableNumber("Buddy Balance Motor P", kDefaultMotorP, unifiedPIDController::setP);
+    tunerNumI = new TunableNumber("Buddy Balance Motor I", kDefaultMotorI, unifiedPIDController::setI);
+    tunerNumD = new TunableNumber("Buddy Balance Motor D", kDefaultMotorD, unifiedPIDController::setD);
+
+    buddyBalancePosEntry.setString("Docked");
+  }
+
+  public double getDesiredAngle() {
+    return Math.round(AbsoluteEncoder.getPositionRadians(encoder)*100)/100.0;
   }
 
   public boolean getIsDeployed() {
     return isDeployed;
   }
 
-  public void deployBuddyBalance() {
-    activateDeploy.set(kServoDeployedPos);
+  public void deployBuddyBalance() { // Removes servo locking mechanism, releasing gas shocks
+    deployServo1.set(kServoDeployedPos1);
+    deployServo2.set(kServoDeployedPos2);
     isDeployed = true;
+    buddyBalancePosEntry.setString("Deployed");
+    servoPositionEntry.setDouble(deployServo1.get());
+  }
+
+  public void returnServos() { // Returns servos to the docked position
+    deployServo1.set(kServoInitialPos);
+    deployServo2.set(kServoInitialPos);
+    buddyBalancePosEntry.setString("Docked");
+    servoPositionEntry.setDouble(deployServo1.get());
   }
 
   public void retrieveBuddy() { // Used to pick up the buddy robot while the lift is already underneath it
-    rightPIDController.setReference(kBalancedPosition, CANSparkMax.ControlType.kPosition);
-    leftPIDController.setReference(kBalancedPosition, CANSparkMax.ControlType.kPosition);
+    unifiedPIDController.setSetpoint(Units.degreesToRadians(kRetrievedAngle));
+    buddyBalancePosEntry.setString("Raised");
   }
 
   public void releaseBuddy() { // Used to set down the robot
-    rightPIDController.setReference(kDeployedPosition, CANSparkMax.ControlType.kPosition);
-    leftPIDController.setReference(kDeployedPosition, CANSparkMax.ControlType.kPosition);
+    unifiedPIDController.setSetpoint(Units.degreesToRadians(kDeployedAngle));
+    buddyBalancePosEntry.setString("Lowered");
+  }
+
+  public void updateMotors() {
+    rightMotor1.set(MathUtil.clamp(unifiedPIDController.calculate(getDesiredAngle()), -1, 1));
+    rightMotor2.set(MathUtil.clamp(unifiedPIDController.calculate(getDesiredAngle()), -1, 1));
+    leftMotor1.set(MathUtil.clamp(unifiedPIDController.calculate(getDesiredAngle()), -1, 1));
+    leftMotor2.set(MathUtil.clamp(unifiedPIDController.calculate(getDesiredAngle()), -1, 1));
```

Comment:
Run calculate once and save it to variable to use on each. I'm also concerned that the motors won't behave exactly the same, requiring different PID controllers.

Reply:
There is a hex shaft connecting left and right side where the cam is attached.  

Will that smooth out any differences?

Reply:
Probably we should have all follow one motor since they are all connected. Should only call calculate once per cycle though